core/src/input/input.tsx

```diff
@@ -27,6 +28,13 @@ const getClass = (props: InputProps): string => {
 	return styles.join(" ");
 };
 
+const getColorClass = (props: InputProps): string => {
+	const style = props.style ?? Input.styles.outset;
+	const size = props.size ?? Input.sizes.medium;
+	const styles = [s.input, outline.always, style.main, size.typeColor];
+	return styles.join(" ");
+};
+
```

Comment:
This seems to be the same with "getClass"? Maybe we only need a line in "getClass":

if (props.type === "color") styles.push(size.typeColor)